### DISCUSSION TITLE
Relations & Roles

### DISCUSSION THREAD

**Discussion by rectrick** [serverpod/serverpod#923] [Q&A] [answered]
Thank you for this wonderful broadcast, which clarified a lot of things, but the answer to my last question was frustrating. I expected that Serverpod supports the organization of relations between the user and vendor , as does prisma orm. The developer can adjust this module while writing the model. support in the same way? And what is the way you intend to skip this until the updates are available for this? Because this is one of the most important things in any application or site is relationships.

Example  [screen](https://prnt.sc/XJ8GtIVwRDsC )

**Comment by vlidholt** [comment, marked as answer]
What you are describing in the screen shot can already be done with Serverpod. You can reference enums or any other serializable classes from other classes.

You can also use the `parent` property to make a reference to another table.

```yaml
class: Employee
fields:
  name: String
  company: int, parent=company
```

**Reply by rectrick** [reply]
Thank you, and I am waiting for you to support this with a video showing the validation between User, Admin, and Manager in their powers, because there are no video sources, and this is important because I am interested in supporting this library to build a large project with it, and I want to understand everything in it, as I am working on an application for audio and video montage, and the program allows the client and the artist to enter into a session And attending the process of music production or montage and participating in the recording with the features of Real Time, so there are many video sources that provide safety for me and others so that he does not feel afraid of the tool if he encounters problems, as I also want to explain courses and support Full Stack in Egypt in Dart language and exploit your great tools in producing new generations that make Complete projects.. I strongly support you, and I hope you support us with more videos, even if the videos are very short, but they contain tricks

**Reply by brajas-quinnox** [reply]
Documentation over videos(since they get outdated faster).
